uniprot-config/src/main/java/org/uniprot/store/config/returnfield/model/ReturnField.java

```diff
@@ -34,6 +40,7 @@ public class ReturnField implements Field, Serializable {
     private String groupName;
     private Boolean isDatabaseGroup = false;
     private Boolean isRequiredForJson = false;
+    private Boolean isMultiValueCrossReference = false;
```

Comment:
I do not think so, because we set it by default as false. (To be honest I do not know why we are using the Wrapper boolean, I just followed the others Boolean attributes). I am going to try to change it and test to check if we have any impact

Reply:
The reason is is our return-field json loader and also json schema. So, I am aborting this change, because it will not be simple